src/options/opt_shortcuts.cpp

```diff
@@ -96,16 +96,16 @@ void OptionsTabShortcuts::applyOptions()
     OptShortcutsUI *d       = static_cast<OptShortcutsUI *>(w);
     PsiOptions *    options = PsiOptions::instance();
 
-    int                 toplevelItemsCount = d->treeShortcuts->topLevelItemCount();
-    int                 shortcutItemsCount;
-    int                 keyItemsCount;
-    QTreeWidgetItem *   topLevelItem;
-    QTreeWidgetItem *   shortcutItem;
-    QTreeWidgetItem *   keyItem;
-    QString             optionsPath;
-    QString             comment;
-    QList<QString>      children;
-    QList<QKeySequence> keys;
+    int                                  toplevelItemsCount = d->treeShortcuts->topLevelItemCount();
+    int                                  shortcutItemsCount;
+    int                                  keyItemsCount;
+    QTreeWidgetItem *                    topLevelItem;
+    QTreeWidgetItem *                    shortcutItem;
+    QTreeWidgetItem *                    keyItem;
+    QString                              optionsPath;
+    [[maybe_unused]] QString             comment;
```

Comment:
Looks like perfect candidates for removal

Reply:
removed